pages/content/amp-dev/documentation/guides-and-tutorials/learn/spec/amphtml.md

```diff
@@ -227,6 +227,11 @@ HTML tags can be used unchanged in AMP HTML. Certain tags have equivalent custom
     <td width="30%">svg</td>
     <td>Most SVG elements are allowed.</td>
   </tr>
+  
+  <tr>
+    <td width="30%">canvas</td>
+    <td>Canvas element is only allowed when enclosed within an <a href="https://amp.dev/documentation/components/amp-script">amp-script</a> block.</td>
```

Comment:
Content LGTM! But can we update the link to markdown format and `amp.dev` grow linking such as this one:
` [amp-bind]({{ g.doc('/content/amp-dev/documentation/components/reference/amp-bind.md', locale=doc.locale).url.path }}?format=websites) ` in [amp-script documentation](https://github.com/ampproject/amp.dev/blob/6c59b50e67406e5a8064938dafc688c17aa9f433/pages/content/amp-dev/documentation/guides-and-tutorials/develop/custom-javascript.md)